script/console

```diff
@@ -23,6 +23,20 @@ if [ -n "$1" ]; then
 else
   # no argument provided, so just run the local console in the development
   # environment. Ensure the application is up to date first.
-  script/update.sh
-    echo "local console not yet implemented"
-fi
\ No newline at end of file
+  # script/update.sh
+
+  echo "To run a local console, please execute the following command:"
+  echo "    source ~/python3.6/bin/activate"
```

Comment:
So, that's what my note was in the description of the PR, I tried to make it work, but I was not successful. The goal for the scripts is to try and standardize on them, but for this particular project, there isn't a concept of a local "console" yet. Once we start merging in the Zappa stuff, they're going to be more relevant.